### ISSUE TITLE
Add accessible-focus mixin

### ISSUE THREAD

**Issue by josephgknox** [SU-SWS/decanter#114]
An accessible-focus mixin was created for a client project and would be useful to host in Decanter.

The mixin code is as follows:

@charset "UTF-8";

//
// Accessibile Focus
// For applying an outline to the focus state of an element or component.
// Helps improve accessibility.
//

@mixin accessible-focus {
  outline: 2px dotted $color-sky;
}

**Comment by kgcreative** [comment]
Focusable elements should not be overriden from defaults. I have in the past made exceptions for `outline-offset` in cases where the outline feels too claustrophobic, but keyboard and accessible technology users rely on consistent outlines across their devices. When we change outline style, we often interfere with users expectations.

**Comment by sherakama** [comment]
This is ringing a bell. I'm pretty sure we have had a chat with Jiatyan about this in the past. @josephgknox It would be a good idea to bounce this off of her. 
@kgcreative Thanks dood! Hope all is well.

**Comment by josephgknox** [comment]
@kgcreative 👋

For sure. I will be sure to check in with Jiatyan before implementing this in Decanter.

**Comment by kgcreative** [comment]
👋hi guys! All is well!

My recommendation would be to make sure there is no focus reset in any boilerplate. Sometimes there is a `:focus { outline: none }` property that gets added to either `body` or `html` that resets the focus state. I believe none of our resets do that, and if there are present, my recco would be to find those and removing them, rather than finding a focus state and normalizing it

**Comment by sherakama** [comment]
Of for sure: http://www.outlinenone.com/
